tests/modules/snpsift/split/main.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SNPSIFT_SPLIT } from '../../../../modules/snpsift/split/main.nf' addParams( options: [:] )
+
+workflow test_snpsift_split_base {
+
+    input = [ [ id:'test'], // meta map
+            file(params.test_data['homo_sapiens']['illumina']['test_rnaseq_vcf'], checkIfExists: true) ]
+
+    SNPSIFT_SPLIT ( input )
+}
+
+workflow test_snpsift_split_mult {
+
+    input = [ [ id:'test'], // meta map
+            file("https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/variants/variants.vcf", checkIfExists: true) ]
```

Comment:
the testdata we use for the modules should be in the nf-core/test-dataset modules branch. Can you use some of the existing testdatasets for this test?

Reply:
Ok yes in that case some test data should be added to the modules branch, preferably with some of the files that exist. I see how that is difficult with the data we have since they are all on chr22. Hm, maybe concast two of the vcf files and replace the chromosome name of one?

Reply:
okay, i can prepare another PR in the test data then

Reply:
Would it also be an option to add the `vcf` test data from the `epitopepredition` branch on the `modules` branch?